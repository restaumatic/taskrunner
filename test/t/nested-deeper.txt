echo "Level 1"
taskrunner -n nested sh -c '
  echo "Level 2"
  taskrunner -n nested_again echo "Level 3"
'
echo "Toplevel task again"
