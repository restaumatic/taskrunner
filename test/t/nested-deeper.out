-- output:
[toplevel] stdout | Level 1
[nested] stdout | Level 2
[nested_again] stdout | Level 3
[toplevel] stdout | Toplevel task again
